arrays-intermediate.js

```diff
@@ -157,15 +177,23 @@ Task 7:
    Hint: Use the .map method and separator operator
 **************************************************************/
 function updateGrades(curve, students) {
-  //TODO: Add your code here
+
+  const updatedGrades = students.map((std)=>{
+    if (std.type !=="nerd"){
+    return { ...std, grade:std.grade+curve};
+    }
+    else{return std}
```

Comment:
The "nerds" grade should also be changed, their grades should be subtracted with the curve.